test/oli/delivery_test.exs

```diff
@@ -89,5 +89,24 @@ defmodule Oli.DeliveryTest do
       assert delivery_setting.id == updated_delivery_setting.id
       assert updated_delivery_setting.collab_space_config.status == :archived
     end
+
+    test "maybe_update_section_contains_explorations/1 update contains_explorations field" do
+      {:ok,
+       project: _project,
+       section: section,
+       page_revision: _page_revision,
+       other_revision: other_revision} = project_section_revisions(%{})
```

Comment:
NIT: If we're not using `project` and `page_revision` we can remove them from the map

Reply:
Here, I think that should match the full map anyway because otherwise it doesn't match what the helper function I'm using returns.